source/03_Dataframes.Rmd

```diff
@@ -758,7 +758,7 @@ arrange(mdf, Testscore) # dplyr muss geladen sein
 In der Funktion `arrange` geben wir als erstes Argument den zu
 sortierenden `data.frame` an. Darauf folgen -- mit Komma separiert --
 alle Spalten nach denen wir sortieren wollen (hier erst mal nur der
-Testscore). Standardmäßig sortiert `arrange` *aufsteigend*, wenn wir
+Testscore). Standardmäßig sortiert `arrange` *aufsteigend*; wenn wir
```

Comment:
Hier lohnt es sich ggf, desc() zu erklären. Das ist nützlich, wenn man MEHRERE Spalten absteigend sortieren will. Das geht mit dem Minus in der Form nicht.
arrange(test, desc(Item1, ID))